webpack.config.js

```diff
@@ -1,34 +1,40 @@
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
-const path = require('path')
-const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
-const nodeExternals = require('webpack-node-externals');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const path = require('path');
+const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const { CheckerPlugin } = require('awesome-typescript-loader');
 
 module.exports = {
-    mode: "development",
-    devtool: "inline-source-map",
-    entry: "./src/index.ts",
-    target: 'node',
+    mode: 'production',
+    devtool: 'inline-source-map',
+    entry: './src/index-browser.ts',
     output: {
-        path: path.resolve(__dirname, './build'),
-        filename: 'bundle.js',
+        // filename: "bundle.js"
+        path: path.resolve(__dirname, './dist'),
```

Comment:
Why switch output directories?

Reply:
We used `build folder` before to test.
Currently. we use `dist folder` for npm and use webpack to minify the package.
It is better the min.js is in `dist`.
So I changed.